puppet/zulip/manifests/postgres_appdb_tuned.pp

```diff
@@ -3,6 +3,22 @@
 class zulip::postgres_appdb_tuned {
   include zulip::postgres_appdb_base
 
+  case $::osfamily {
+    'debian': {
+      $postgres_conf_file = "${zulip::postgres_appdb_base::postgres_confdir}/postgresql.conf"
+      $postgres_restart = "pg_ctlcluster ${zulip::base::postgres_version} main restart"
```

Comment:
I think it'd be better to define these variables in n`postgres_common` with the others, and then use them here as needed.

Reply:
That will result in circular dependency. In the `postgres_conf_file` var, it uses `zulip::postgres_appdb_base::postgres_confdir`, and `postgres_appdb_base` already includes `postgres_common`.

Reply:
Besides, the variables are only used inside `postgres_appdb_tuned.pp`. Might as well just define them right where they are needed.